hibernate-spatial/src/main/java/org/hibernate/spatial/HibernateSpatialConfiguration.java

```diff
@@ -0,0 +1,76 @@
+package org.hibernate.spatial;
+
+import org.geolatte.geom.codec.db.oracle.ConnectionFinder;
+
+import java.io.Serializable;
+
+import org.jboss.logging.Logger;
+
+/**
+ * A global configuration object that is is used by
+ * some Dialects during construction.
+ *
+ * @author Karel Maesen, Geovise BVBA
+ *         creation-date: 8/16/13
+ */
+public class HibernateSpatialConfiguration implements Serializable {
+
+	private static final HSMessageLogger LOG = Logger.getMessageLogger(
+			HSMessageLogger.class,
+			HibernateSpatialConfiguration.class.getName()
+	);
+	private Boolean isOgcStrict = Boolean.TRUE;
+	private ConnectionFinder connectionFinder;
+
+	/**
+	 * Holds the configuration for Hibernate Spatial dialects.
+	 */
+	public HibernateSpatialConfiguration() {
+	}
+
+	/**
+	 * Creates a Configuration for Hibernate spatial
+	 *
+	 * @param ogcStrict true for OGC Strict mode
+	 * @param connectionFinder the fully-qualified Class name for the {@code ConnectionFinder}
+	 */
+	public HibernateSpatialConfiguration(Boolean ogcStrict, ConnectionFinder connectionFinder) {
+		if ( ogcStrict != null ) {
+			this.isOgcStrict = ogcStrict;
+			LOG.info( String.format( "Setting OGC_STRICT mode for Oracle Spatial dialect to %s.", ogcStrict ) );
+		}
+		if ( connectionFinder != null ) {
+			this.connectionFinder = connectionFinder;
+			LOG.info(
```

Comment:
Again, wrt logging... by convention we say that everything info and higher should be defined on a MessageLogger.  The reason being that MessageLogger messages are translated, and since these levels have a higher likelihood of being seen by users, that makes sense.  TRACE and DEBUG are more intended for developers.